recipes/torchsde/meta.yaml

```diff
@@ -22,14 +22,17 @@ requirements:
   run:
     - python
     - blist
-    - numpy >=1.17.0
-    # - pytorch::pytorch >=1.6.0
+    - boltons >=20.2.1
+    - numpy >=1.19.1
+    - trampoline >=0.1.2
+    - scipy >=1.5.2
+  run_constrained:
+    - torch >=1.6.0
```

Comment:
where is the torch package?

Pytorch releases a package called `pytorch` not `torch`.
https://pytorch.org/